packages/cozy-stack-client/src/ContactsCollection.spec.js

```diff
@@ -36,6 +36,16 @@ describe('ContactsCollection', () => {
     jest.spyOn(console, 'warn').mockImplementation(() => {})
   })
 
+  it('should handle undefined selector', async () => {
+    const { col, stackClient } = setup()
+    await col.find()
+    expect(stackClient.fetchJSON).toHaveBeenCalledWith(
+      'POST',
+      '/data/io.cozy.contacts/_find',
+      expect.objectContaining({ selector: undefined })
```

Comment:
Does that mean we actually send `selector: undefined` to the stack, or the selector attribute is not there?

Reply:
not if we have an partial index with combine with selector. I didn't define partialFilter for this test so it's return undefined

Reply:
The test should be : 
`it('call find route if there is no selector')`

and we should just test that `_find` has been called. We don't need to test the object